packages/runtime/src/widgets/Icon.tsx

```diff
@@ -30,7 +33,7 @@ export const Icon: React.FC<IconProps> = ({ onTap, ...props }) => {
   return (
     <IconComponent
       className={values?.styles?.names}
-      onClick={(): unknown => onTapActionCallback?.callback()}
+      onClick={(): unknown => onTapActionCallback?.callback({ ...scope })}
```

Comment:
Scope should be injected by the action, not passed in. Which action is missing the scope?

Reply:
It's executeCode that is not injecting the custom scope, not the onTap trigger. This should be fixed on the latest main.